crates/release_plz/Cargo.toml

```diff
@@ -14,13 +14,14 @@ categories = ["command-line-utilities", "development-tools"]
 workspace = true
 
 [features]
-default = ["docker-tests"]
+default = ["docker-tests", "cargo-vendored-libs"]
```

Comment:
let's call this feature `all-static`, to follow `cargo` convention :)

Reply:
Fixed.